wiki/Rules/en.md

```diff
@@ -6,7 +6,7 @@ These rules are the basis from which we help keep the osu! community a fun and w
 Community Rules
 ---------------
 
-1.  **Each player may only have *ONE* account at any point of time.** This account is you. It is not anyone else - not your brother, your mother, your sister, your friend - it is YOU. Don't share your account with anyone else.
+1.  **Each player may only have *ONE* account, ever.** The *first* account you make during account registration is the account you will have in *osu!* forever. This account is YOU. It is not anyone else - not your brother, your mother, your sister, your friend - it is YOU. Don't share your account with anyone else. You don't get to make a new one if you lose access to it. Keep it safe.
```

Comment:
You don't get into You mustnt?

Reply:
musn't implies it is a choice they can, but should not consider

'don't get' is conversational for 'you cannot do this. ever.'